frontend-react/src/network/QueryClients.ts

```diff
@@ -2,7 +2,14 @@ import { QueryClient } from "@tanstack/react-query";
 
 export const appQueryClient = new QueryClient({
     defaultOptions: {
-        queries: { suspense: true, useErrorBoundary: true },
+        queries: {
+            suspense: true,
+            useErrorBoundary: true,
+            retry: false,
```

Comment:
No more retries + we're okay with stale queries since our data isn't up-to-the-minute accurate anyway.  The fewer fetches, the better 🙏

Reply:
This is the _real_ star of the show, here 😍

Reply:
Seriously!  I haven't been so excited for configuration since......ever 😆